src/test_utils.erl

```diff
@@ -153,6 +156,19 @@ to_var({var, Name}) ->
     "'" ++ string:replace(erlang:atom_to_list(Name), "$", "u").
 
 
+named(Ref, Args) ->
+  % Use the dummy '.' file as the module for testing purposes
+  {named, ast:loc_auto(), {ty_ref, '.', Ref, length(Args)}, Args}.
+
+extend_symtab(Def, Scheme) ->
+  extend_symtab(Def, Scheme, symtab:empty()).
```

Comment:
There is now symtab:std_symtab() that will include the standard library. I don't know whether this is relevant for the tests.